EZP-27506 toggle locations

mcurran-inviqa opened the pull request.

maxbaldanza commented on src/bundle/Controller/LocationController.php:
```diff
@@ -122,6 +125,50 @@ public function updateDefaultSortOrderAction(
+
+            $locationService->hideLocation($location);
+            /** @Desc("The Location #%id% is now hidden") */
+            $message = $translator->trans(
```
Are these messages used or is it still a WIP?

maxbaldanza commented on src/bundle/Controller/LocationController.php:
```diff
@@ -122,6 +125,50 @@ public function updateDefaultSortOrderAction(
+            $visibilityForm->handleRequest($request);
+
+            if (!$visibilityForm->isValid()) {
+                throw new \Exception('Invalid form submission.');
```
Does this need to return a message rather than throwing an exception as below?

maxbaldanza reviewed (commented):
Looks good, just a few comments about some of the messaging stuff

mcurran-inviqa committed:
EZP-27506 toggle locations

- Adds location visibility form
- Adds visibility fields to actions form for populating location visibility form
- Adds handler for visibility toggle
